test/support/spec.rb

```diff
@@ -233,19 +255,23 @@ def as_user
   yield
 end
 
+def as_superuser
+  nil
+end
+
 def little_endian
   yield
 end
 
 def big_endian
 end
 
-@tmp_uniq_id = 0
+TMP_UNIQ_ID = 0
```

Comment:
Can you talk about this change from ivar to constant? MRI will warn on modifying constants, and we should too.

Reply:
Sorry about this.  This change broke the CI and ruby tests though it worked in Natalie on my end.
I think I was trying to fix some behavior and was trying to model off of what goes on in Mspec, but IIRC there they are using a constant with a String value and calling `.succ!` on it which is a bit of trickery.
Regardless, this edit has been reverted with the latest push.

Reply:
Ok, makes sense!